srcpkgs/m1n1/files/m1n1-kernel-hook.confd

```diff
@@ -0,0 +1,10 @@
+CONFIG=/etc/m1n1.conf
+# can be either 1=kernel or 2=uboot (default)
+PAYLOAD=2
```

Comment:
why can't we just use "kernel" or "uboot"? the number seems like an unnecessary indirection

Reply:
Sure, that's also possible.